code/game/objects/items/weapons/switchtool.dm

```diff
@@ -355,6 +355,8 @@
 		item_state = "uvsoap"
```

Comment:
presumably these are the ```item_states``` of the deployed item.
so why isn't this huge block of ifs just:
```
if(deployed)
    item_state = deployed.item_state
```